templates/_linux.yaml

```diff
@@ -12,6 +12,7 @@
       /objects[0].spec.template.spec.domain.devices.disks
       /objects[0].spec.template.spec.volumes
       /objects[0].spec.template.spec.networks
+      /objects[0].spec.template.spec.ioThreadsPolicy
```

Comment:
```suggestion
```
The global VM policy should stay shared. The user can select diffferent policy per disk (like a dedicated thread for specific disk only).

Reply:
But why Stu wrote this feature (different policy?) Do we use it anywhere?

Reply:
There are technically 3 modes. "shared", "auto" and "none". The default should probably be to omit the IOThreadsPolicy setting and thus disable it?

The feature @MarSik is referring to is a per-disk override `dedicatedIOThread` that will allocate a thread for a single disk regardless of what the rest of the template is using.

Reply:
That is exactly what I had in mind. Thanks @stu-gott for clarification.